docs/src/man/basics.md

```diff
@@ -0,0 +1,1388 @@
+   1 │     1     22  female      2  own      little           moderate         ⋯
+                                                               3 columns omitted
+
+julia> german[3, :] # this produces a DataFrameRow which is treated as 1-dimensional object similar to a NamedTuple
+DataFrameRow
+ Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking account ⋯
+     │ Int64  Int64  String  Int64  String   String           String           ⋯
+─────┼──────────────────────────────────────────────────────────────────────────
+   3 │     2     49  male        1  own      little           NA               ⋯
+                                                               3 columns omitted
+```
+
+We can get a single cell or element with the same syntax to get an element of an array:
+```jldoctest dataframe
+julia> german[4, 4]
+2
+```
+
+or to get a new `DataFrame` that is subset of rows and columns:
+```jldoctest dataframe
+julia> german[4:5, 4:5]
+2×2 DataFrame
+ Row │ Job    Housing
+     │ Int64  String
+─────┼────────────────
+   1 │     2  free
+   2 │     2  free
+```
+
+Assignment of a scalar to a data frame can be done in ranges using broadcasting:
+```jldoctest dataframe
+julia> german[1:2, 1:2] .= 1
+2×2 SubDataFrame
+ Row │ id     Age
+     │ Int64  Int64
+─────┼──────────────
+   1 │     1      1
+   2 │     1      1
+
+julia> german
+1000×10 DataFrame
+  Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking accoun ⋯
+      │ Int64  Int64  String  Int64  String   String           String          ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     1      1  male        2  own      NA               little          ⋯
+    2 │     1      1  female      2  own      little           moderate
+    3 │     2     49  male        1  own      little           NA
+    4 │     3     45  male        2  free     little           little
+    5 │     4     53  male        2  free     little           little          ⋯
+    6 │     5     35  male        1  free     NA               NA
+    7 │     6     53  male        2  own      quite rich       NA
+    8 │     7     35  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮       ⋮            ⋮                ⋮        ⋱
+  994 │   993     30  male        3  own      little           little          ⋯
+  995 │   994     50  male        2  own      NA               NA
+  996 │   995     31  female      1  own      little           NA
+  997 │   996     40  male        3  own      little           little
+  998 │   997     38  male        2  own      little           NA              ⋯
+  999 │   998     23  male        2  free     little           little
+ 1000 │   999     27  male        2  own      moderate         moderate
+                                                  4 columns and 985 rows omitted
+```
+
+The indexing syntax can also be used to select rows based on conditions on variables:
```

Comment:
again - the section promised to discuss single column operations

Reply:
you give an example of something else than single column and this section is about single column selection